spec/models/manageiq/providers/kubernetes/container_manager_spec.rb

```diff
@@ -47,4 +47,40 @@
       expect(job.zone).to eq("default")
     end
   end
+
+  context "Proxy Configuration" do
```

Comment:
@chrisarcand ```kubeclient``` just passes this parameter to ```rest-client``` which will pass it to ```net::http```. The end goal is to have the http request be routed to the proxy.